dht/murmur3_partitioner.cc

```diff
@@ -78,13 +70,7 @@ murmur3_partitioner::get_token(bytes_view key) const {
 
 token
 murmur3_partitioner::get_token(uint64_t value) const {
```

Comment:
Also, maybe `get_token` should take `int64_t` now. I never understood why it takes `uint64_t`.

Reply:
Probably but it's out of the scope of this patchset.